modules/nf-core/cellsnp/modea/tests/main.nf.test

```diff
@@ -0,0 +1,91 @@
+// nf-core modules test cellsnp/modea
+nextflow_process {
+
+    name "Test Process CELLSNP_MODEA"
+    script "../main.nf"
+    process "CELLSNP_MODEA"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "cellsnp"
+    tag "cellsnp/modea"
+    tag "samtools/index"
+
+    test("genotyping") {
+        setup { 
+            run("SAMTOOLS_INDEX") {
+                script "../../../samtools/index/main.nf"
+                process {
+                    """
+                    input[0] = [
+                    [ id:'sample1' ],
+                    file(params.modules_testdata_base_path + '/genomics/homo_sapiens/demultiplexing/chr21.bam', checkIfExists: true) ]
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """ 
+                
+                input[0] = [[ id: 'sample1'], 
+                            file(params.modules_testdata_base_path + '/genomics/homo_sapiens/demultiplexing/chr21.bam', checkIfExists: true),
+                            SAMTOOLS_INDEX.out.bai.map{it[1]},
+                            file(params.modules_testdata_base_path + /genomics/homo_sapiens/demultiplexing/donor_genotype_chr21.vcf', checkIfExists: true),
```

Comment:
```suggestion
                            file(params.modules_testdata_base_path + '/genomics/homo_sapiens/demultiplexing/donor_genotype_chr21.vcf', checkIfExists: true),
```

Reply:
@SPPearce I tried your suggestion but didnt pass the tests due to the error ` Not a valid path value type: groovyx.gpars.dataflow.DataflowVariable (DataflowVariable(value=null))` . It seems like I was trying to add a channel to the input channel. So i go back to the previous version now